Gems/AtomLyIntegration/AtomBridge/Code/Source/AtomDebugDisplayViewportInterface.h

```diff
@@ -176,7 +176,7 @@ namespace AZ::AtomBridge
         void DrawWireHemisphere(const AZ::Vector3& pos, const AZ::Vector3& axis, float radius) override;
         void DrawWireDisk(const AZ::Vector3& pos, const AZ::Vector3& dir, float radius) override;
         void DrawBall(const AZ::Vector3& pos, float radius, bool drawShaded) override;
-        void DrawDisk(const AZ::Vector3& pos, const AZ::Vector3& dir, float radius) override;
+        void DrawDisk(const AZ::Vector3& pos, const AZ::Vector3& dir, float radius, bool drawShaded = true) override;
```

Comment:
I think these default values needs to be in the interface header (EntityDebugDisplayBus.h), otherwise it only applies to people that directly using this class as the interface.